### PR TITLE
UPD fix patients list styling

### PR DESCRIPTION
**Pull request by mateuszkrasucki**
- Fixes patients list (both general patient list and waitlist)
  styling according to guidelines:
    - Border should start after avatar/image.
    - Proper alignment.
    - Main complain in the waitlist.
    - Nationality displayed with code title, not code.

### CODE REVIEW

File: frontend/shared/containers/codes/index.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            return <span>Failed to fetch title!</span>
+        }
+
+        return <span>{this.state.title}</span>
+    }
+}
+
+CodeTitle = connect(state => ({}), { fetchCode })(CodeTitle)
+
+export { CodeTitle }
```
Comment (dominikznidar) [inline frontend/shared/containers/codes/index.js]: Would probably make more sense to name the `codeTitles/index.js` and export it with `export default`.

File: frontend/shared/containers/patient/index.js [hunk 2]
```diff
@@ -32,3 +48,6 @@ export default ({ data, style, big }) => {
         </div>
     )
 }
+
+export { PatientImage, PatientCard }
```
[the remark this answers is not in the document]
Comment (mateuszkrasucki) [reply frontend/shared/containers/patient/index.js]: I don't think so - I was thinking about preventing exponential growth of containers dir. Maybe I'll make them component per file but under patient folder?

File: frontend/shared/containers/patient/index.js [hunk 3]
```diff
@@ -18,7 +30,11 @@ export default ({ data, style, big }) => {
 
     return (
         <div className={classnames("patientCard", { big })}>
-            <img src={PersonPlaceholder} alt="" />
+            {withoutImage ? null : (
+                <div className={classnames("patientImage", { big })}>
+                    <img src={PersonPlaceholder} alt="" />
+                </div>
```
[the remark this answers is not in the document]
Comment (mateuszkrasucki) [reply frontend/shared/containers/patient/index.js]: 👍

File: frontend/shared/containers/codes/index.js (hunk 1 above)
Comment (mateuszkrasucki) [reply frontend/shared/containers/codes/index.js]: ok, related to your first comment. What about codes/title/index.js?

File: frontend/shared/containers/patient/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (dominikznidar) [reply frontend/shared/containers/patient/index.js]: true, we use the index.js in this project. What about `patient/card[.js]` and `patient/image[.js]`?

[the remark this answers is not in the document]
Comment (mateuszkrasucki) [reply frontend/shared/containers/patient/index.js]: yeah, let's do it like that, like with login/privateRoute

File: frontend/shared/containers/codes/index.js (hunk 1 above)
Comment (mateuszkrasucki) [reply frontend/shared/containers/codes/index.js]: will do it as codes/title.[js]

**Comment by mateuszkrasucki** [comment]
@dominikznidar issues were resolved following comments.

**Pull request merged**